docs/docs/upgrading.md

```diff
@@ -5,6 +5,15 @@ description: >-
   for Pomerium. Please read it carefully.
 ---
 
+# Since 0.14.0
+
+## New
```

Comment:
@desimone @alexfornuto I stubbed out new features that could use highlighting. 

1. Are there others that make sense?
2. We need to document the new `routes` syntax and PPL in OSS and link from here.  Can be a FUP once that's ready.

Reply:
I think we'll have to merge #2390 before linking to the new PPL / Routes sections which are currently WIP.